public/celestia-node.sh

```diff
@@ -184,100 +157,39 @@ echo    # move to a new line
-            ln -s "$GOBIN/celestia" "$TEMP_DIR/celestia"
-            echo "Symbolic link created in $TEMP_DIR" | tee -a "$LOGFILE"
-            echo ""
-            # Check if GOBIN is in PATH
-            if [[ ":$PATH:" != *":$GOBIN:"* ]]; then
-                echo "NOTE: $GOBIN is not in your PATH. You may want to add it by adding this line to your ~/.bashrc or ~/.zshrc:"
-                echo "export PATH=\$PATH:$GOBIN"
-                echo ""
-            fi
-            echo "You can now run celestia from anywhere (if $GOBIN is in your PATH)." | tee -a "$LOGFILE"
             ;;
         2)
-            # Install to /usr/local/bin
             sudo mv "$TEMP_DIR/celestia" /usr/local/bin/
             echo "Binary moved to /usr/local/bin" | tee -a "$LOGFILE"
-            # Create a symbolic link in the temporary directory
-            ln -s /usr/local/bin/celestia "$TEMP_DIR/celestia"
-            echo "Symbolic link created in $TEMP_DIR" | tee -a "$LOGFILE"
-            echo ""
-            echo "You can now run celestia from anywhere." | tee -a "$LOGFILE"
             ;;
         3)
-            # Keep in current directory
             chmod +x "$TEMP_DIR/celestia"
             echo "Binary kept in $TEMP_DIR" | tee -a "$LOGFILE"
-            echo "You can run celestia from this directory using ./celestia" | tee -a "$LOGFILE"
             ;;
         *)
-            echo ""
-            echo "Invalid choice. The binary remains in $TEMP_DIR" | tee -a "$LOGFILE"
-            chmod +x "$TEMP_DIR/celestia"
-            echo "You can run celestia from this directory using ./celestia" | tee -a "$LOGFILE"
+            echo "Invalid choice. Exiting." | tee -a "$LOGFILE"
+            exit 1
             ;;
     esac
 else
     case $REPLY in
         1)
-            # Install to /usr/local/bin
             sudo mv "$TEMP_DIR/celestia" /usr/local/bin/
             echo "Binary moved to /usr/local/bin" | tee -a "$LOGFILE"
-            # Create a symbolic link in the temporary directory
-            ln -s /usr/local/bin/celestia "$TEMP_DIR/celestia"
-            echo "Symbolic link created in $TEMP_DIR" | tee -a "$LOGFILE"
-            echo ""
-            echo "You can now run celestia from anywhere." | tee -a "$LOGFILE"
             ;;
         2)
-            # Keep in current directory
             chmod +x "$TEMP_DIR/celestia"
             echo "Binary kept in $TEMP_DIR" | tee -a "$LOGFILE"
-            echo "You can run celestia from this directory using ./celestia" | tee -a "$LOGFILE"
             ;;
         *)
-            echo ""
-            echo "Invalid choice. The binary remains in $TEMP_DIR" | tee -a "$LOGFILE"
-            chmod +x "$TEMP_DIR/celestia"
-            echo "You can run celestia from this directory using ./celestia" | tee -a "$LOGFILE"
+            echo "Invalid choice. Exiting." | tee -a "$LOGFILE"
+            exit 1
             ;;
     esac
 fi
 
-echo ""
```

Comment:
This is meant to give someone instructions on how to see the help menu and how to interact with the binary off the start. I think it is best to also leave this. Or at least work it into a separate PR so we can get the curl update in.

Reply:
I think it is best that this stays. However, if the symlink isn't needed, would understand. but it is best to address in separate issue and new PR